src/schema/collection/resolvers.ts

```diff
@@ -14,6 +14,7 @@ const normalizeCollection = (collection) => {
     'socialMediaDescription',
     'curatedEventsTitle',
     'eventListQuery',
+    'hobbyListQuery',
```

Comment:
should we use the name "hobby" all across the project or course that has already been used?

Reply:
Huh, I thought hobby list and course are different terms here. 😅  If they are the same, I think it's better to rename it to courseListQuery for consistency

Reply:
I think they are same :D

Reply:
At least I have been using course term everywhere :)